Sources/Subs-Admin.php

```diff
@@ -1221,13 +1221,16 @@ function updateSettingsFile($config_vars, $keep_quotes = null, $rebuild = false)
 	// It's important to do the numbered ones before the named ones, or messes happen.
 	uksort($substitutions, function($a, $b) {
 		if (is_int($a) && is_int($b))
-			return $a > $b;
+			return $a > $b ? 1 : 0;
 		elseif (is_int($a))
 			return -1;
 		elseif (is_int($b))
 			return 1;
 		else
-			return strcasecmp($b, $a);
+		{
+			$res = strcasecmp($b, $a);
+			return $res > 0 ? 1 : ($res < 0 ? -1 : 0);
+		}
```

Comment:
strcasecmp() returns an int compatible with uksort()... can you reverse this change?

Reply:
I was thinking that uksort needed -1, 0 or 1 for return, but the docs says it can be any int.

I changed the line above because $a < $b should programmatically be -1 not 0.  0 should be for when they are equal